packages/content-conformance/src/__tests__/cli.test.ts

```diff
@@ -24,7 +24,8 @@ describe('Content-conformance CLI', () => {
     )
 
     expect(res).toMatchInlineSnapshot(`
-"Configuring content conformance runner...
+"[ 'content-check' ]
```

Comment:
👀 These snapshot updates don't look quite right, was this from a local change?

Reply:
oh no! That was from our code sess when we were checking the default command